spec/system/managing_topics_spec.rb

```diff
@@ -21,4 +21,18 @@
     expect(current_path).to eq topics_path
     expect(page).to have_content 'tell us a story'
   end
+
+  scenario 'viewing a topic' do
+    visit topics_path
+    click_on('What is your favourite colour')
+    expect(current_path).to eq topic_path(@topic.id)
+    expect(page).to have_content('Topic: 1')
+  end
+
+  scenario 'when deleting a topic' do
+    visit topic_path(@topic.id)
```

Comment:
No need to call id on the topic. The following is enough:

```
visit topic_path(topic)
```

(Also, you `let` blocks in specs, rather than instance variables)